exabyte_api_client/endpoints/jobs.py

```diff
@@ -154,3 +154,17 @@ def get_presigned_urls(self, id_, files):
         data = json.dumps({"files": files})
         response = self.request('POST', '/'.join((self.name, id_, "presigned-urls")), data=data, headers=self.headers)
         return response["presignedURLs"]
+
+    def list_files(self, id_):
+        """
+        Returns a list of job files.
+
+        Args:
+            id_ (str): job ID.
+
+        Returns:
+            list: [{ "key" : str, "size" : int, "bucket" : str, "region" : str,
+                     "provider" : str, "lastModified" : int, "name" : str, "signedURL" : str }]
+        """
+        response = self.request('GET', '/'.join(('jobs', id_, 'files')), headers=self.headers)
+        return response
```

Comment:
https://stackoverflow.com/questions/36043061/automatically-add-newline-on-save-in-pycharm